code/game/machinery/mind_machine.dm

```diff
@@ -0,0 +1,846 @@
+/obj/machinery/mind_machine
+	name = "Mind Machine"
+	icon = 'icons/obj/mind_machine.dmi'
+	density = 1
+	anchored = 1.0
```

Comment:
anchored is a binary value, drop that .0

Reply:
More like use real boolean constants:

```suggestion
	anchored = TRUE
```